compatibility/README.md

```diff
@@ -2,3 +2,14 @@ This directory contains the infrastructure and test setup for
 cross-version compatibility testing. We make this a separate Bazel
 workspace to make it easier to enforce that we only depend on release
 artifacts.
+
+### How to
+
+Before using this, make sure to always run the scripts that build the
+SDK, namely `./build-release-artifacts.sh` and `./build-ts-libs.sh`.
+These should be run every time there is a relevant change you want
+to be there for this workspace to use.
+
+Any Bazel command can then be used afterwards. `./test.sh` are the
```

Comment:
Note that `./test.sh` will run _all_ tests. We are already up to > 700 iirc so if cashing doesn’t work this is a bad idea. There is `./test.sh --quick` which only runs latest stable vs HEAD and once you copied `.bazelrc` you can also just use `bazel test`.